install_config/cluster_metrics.adoc

```diff
@@ -52,6 +52,17 @@ endif::[]
 
 == Before You Begin
 
+[WARNING]
+====
+If your OpenShift installation was originally performed on a version previous to
+v1.0.8, even if it has since been updated to a newer version, you will need to 
```

Comment:
Is there an example of conditional flags being used in the docs somewhere? I don't think I have come across any before